src/mechanics/interpolation/CellInterpolationFEM.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include <vector>
+#include "mechanics/nodes/NodeSimple.h"
+#include "InterpolationSimple.h"
+#include "CellInterpolationBase.h"
+
+namespace NuTo
+{
+
+class CellInterpolationFEM : public CellInterpolationBase
+{
+public:
+    CellInterpolationFEM(std::vector<NuTo::NodeSimple*> rNodes, const InterpolationSimple& rInterpolation)
+        : mNodes(rNodes)
+        , mInterpolation(rInterpolation)
+    {}
+
+    //! @brief extracts all node values of this element
```

Comment:
Inherited member functions inherit their documentation, no need for this comment.